<launch>
  <!-- Arm 1 -->
  <node pkg="cut_pliers_controller" type="cut_pliers_arm1" name="cut_pliers_arm1" output="screen">
    <param name="port" value="/dev/ttyUSB1"/>
    <param name="baudrate" value="115200"/>
    <param name="timeout" value="100"/>
    <param name="cmd_topic" value="/cmd_cut_pliers_1"/>
    <param name="target_height" value="140"/>
    <param name="target_length" value="-1"/>
    <param name="last_valid_length" value="10"/>
    <param name="claw" value="false"/>
    <param name="allow_retract" value="false"/>
    <param name="log" value="false"/>
  </node>

  <!-- Arm 2 -->
  <!-- <node pkg="cut_pliers_controller" type="cut_pliers_arm2" name="cut_pliers_arm2" output="screen">
    <param name="port" value="/dev/ttyUSB2"/>
    <param name="baudrate" value="115200"/>
    <param name="timeout" value="100"/>
    <param name="cmd_topic" value="/cmd_cut_pliers_2"/>
    <param name="target_height" value="140"/>
    <param name="target_length" value="-1"/>
    <param name="last_valid_length" value="10"/>
    <param name="claw" value="false"/>
    <param name="allow_retract" value="false"/>
  </node> -->
</launch>
